src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java

```diff
@@ -133,6 +136,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             final Marshaller m = ctx.createMarshaller();
             m.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             m.marshal(new Gas(gas), w);
+            if (includeVersion) {
+                w.newLine();
+                w.write("<!-- " + mavenSession.getCurrentProject().getVersion() + "-->");
```

Comment:
What's wrong with having `<version>` as a regular XML element?

Reply:
well nothing specific. I did not want to break the parsing.
But I can add it as a simple tag

Reply:
btw, where would I include it @ppalaga ?
Current XML is 
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<testArtifacts>
</testArtifacts>
```
I can hardly include the `version` tag in the `testArtifacts`
I would need to create a root element or ?

Reply:
I'd vote for having `<version>` tags per `<testArtifact>` which we need elsewhere anyway.

Reply:
I see now